app/server/routes/ra.py

```diff
@@ -0,0 +1,48 @@
+import os
+import sys
+
+from fastapi import APIRouter, Body
+from fastapi.encoders import jsonable_encoder
+
+sys.path.append("../../server")
+from dotenv import load_dotenv
+
+load_dotenv()
+from typing import List
+
+import motor.motor_asyncio
+from fastapi import Body, HTTPException, status
+from fastapi.encoders import jsonable_encoder
+from fastapi.responses import JSONResponse, Response
+from server.models.reactive_actuator import ReactiveActuatorModel
+
+router = APIRouter()
+
+
+from server.database import db
+
+
+@router.post("/", response_description="Add new reactive actuators", response_model=ReactiveActuatorModel)
+async def create_ra(ra: ReactiveActuatorModel = Body(...)):
+    ra = jsonable_encoder(ra)
+    new_ra = await db["ra"].insert_one(ra)
+    created_ra = await db["ra"].find_one({"_id": new_ra.inserted_id})
+    return JSONResponse(status_code=status.HTTP_201_CREATED, content=created_ra)
+
+
+@router.get(
+    "/", response_description="List all reactive actuators", response_model=List[ReactiveActuatorModel]
+)
+async def list_ra():
+    RAs = await db["ra"].find().to_list(1000)
```

Comment:
for all of our routes, I think jonas suggested that we include parameters for how many records we want, food for thought